### PR TITLE
[ACR] Improve deprecation message for `az acr helm`

### PR DESCRIPTION
**Pull request by jasminetMSFT**
**Related command**

`az acr helm`

**Description**

Updated the deprecation message for the az acr helm module to make it clearer and provide better guidance on alternative commands, improving user understanding and experience.

**Testing Guide**
Verified CLI changes manually.
image
image

**History Notes**

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
ACR

File: src/azure-cli/azure/cli/command_modules/acr/commands.py [hunk 1]
```diff
@@ -363,19 +363,28 @@ def load_command_table(self, _):
...
+                                                          message_func=_helm_deprecate_message))
+        g.command('delete', 'acr_helm_delete',
+                            deprecate_info=self.deprecate(redirect="Helm v3 commands",
+                                                          message_func=_helm_deprecate_message))
+        g.command('push', 'acr_helm_push', deprecate_info=self.deprecate(redirect="Helm v3 commands",
+                                                          message_func=_helm_deprecate_message))
+        g.command('repo add', 'acr_helm_repo_add', deprecate_info=self.deprecate(redirect="Helm v3 commands",
+                                                          message_func=_helm_deprecate_message))
+        g.command('install-cli', 'acr_helm_install_cli', is_preview=True, deprecate_info=self.deprecate(redirect="Helm v3 commands",
+                                                          message_func=_helm_deprecate_message))
```
Comment (shizhMSFT) [inline src/azure-cli/azure/cli/command_modules/acr/commands.py:387]: I'm wondering if the `deprecate_info` can be reused.

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Comment by JXavierMSFT** [comment]
LGTM, Great Job @jasminetMSFT!

**Review by yanzhudd — APPROVED** [review]

**Pull request merged**